### ISSUE TITLE
Max acceleration limit

### ISSUE THREAD

**Issue by s265452** [Kinovarobotics/kinova-ros#380]
Hi everyone,
I'm using a Kinova Jaco 2 7 DoF.

is it possibile to know acceleration limits for each Joint?

I also read the technical guide about the actuators at the links below:

https://drive.google.com/file/d/1zGBHD-AuB9Hli0ppUz9s4Ivs5qCP3QUd/view
https://drive.google.com/file/d/1zIWP6OSXGFw63DdBFXqRiZNRYVB08PcC/view

I didn't find these information.

Waiting for your answer.

**Comment by felixmaisonneuve** [comment]
Hi @s265452,

This is not really documented and I had to go look in the code to find something.

According to what I found : 
The base's firmware has an acceleration limit of 200 (degrees/s^2 I assume)
The actuator's firmware has an acceleration limit of 300 (but I am not sure if this value is actually used anywhere).

In either case, each command will have to go through the base first, so I suppose the limit will always be 200.

Jaco's code is getting quite old and it is hard to say with certainty that it is the correct value, but it is the best answer I can give you.

Hope this helps,
Felix
